specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/examples/CreateDwDatabaseByDtuCapacity.json

```diff
@@ -0,0 +1,83 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "Default-SQL-SouthEastAsia",
+    "serverName": "testsvr",
+    "databaseName": "testdw",
+    "api-version": "2017-03-01-preview",
+    "parameters": {
+      "location": "westus",
+      "sku": {
+        "name": "DataWarehouse",
+        "capacity": 9000
```

Comment:
We never refer to DTU in our documentation. So creating DW by DTU is not a valid use case for us.